www/userguide/Doxyfile

```diff
@@ -0,0 +1,12 @@
+PROJECT_NAME           = "CodeChecker"
```

Comment:
Couldn't we use the top level Doxyfile?

Reply:
The top level Doxygen file uses doxygen own header and footer files. This way the html file which the doxygen was generated looks like this:
![cc_doxygen](https://user-images.githubusercontent.com/6695818/31173153-c4b1b0ae-a906-11e7-8146-33ed5c727758.png)
It contains extra style sheets and javascript files. If I'm opening the user guide in the CodeChecker, I'm getting a 404 error because these files doesn't exists in the web root directory.
This is the reason why I used a separate Doxygen file.